mappings/net/minecraft/server/world/ServerEntityManager.mapping

```diff
@@ -0,0 +1,89 @@
+CLASS net/minecraft/class_5579 net/minecraft/server/world/ServerEntityManager
+	FIELD field_27260 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_27261 entityUuids Ljava/util/Set;
+	FIELD field_27262 entityLoader Lnet/minecraft/class_5576;
+	FIELD field_27263 entityDataAccess Lnet/minecraft/class_5571;
+	FIELD field_27264 trackedEntities Lnet/minecraft/class_5570;
+	FIELD field_27265 sectionedEntities Lnet/minecraft/class_5573;
+	FIELD field_27266 entityStore Lnet/minecraft/class_5577;
+	FIELD field_27267 statusByChunk Lit/unimi/dsi/fastutil/longs/Long2ObjectMap;
+	FIELD field_27268 loadingStagesByChunkPos Lit/unimi/dsi/fastutil/longs/Long2ObjectMap;
+	FIELD field_27269 tickingChunkSections Lit/unimi/dsi/fastutil/longs/LongSet;
+	FIELD field_27270 loadingQueue Ljava/util/Queue;
+	METHOD <init> (Ljava/lang/Class;Lnet/minecraft/class_5576;Lnet/minecraft/class_5571;)V
+		ARG 1 entityBaseClass
+		ARG 2 entityLoader
+		ARG 3 entityDataAccess
+	METHOD method_31809 tick ()V
+	METHOD method_31810 ensureEntityChunkDataLoaded (J)V
+		ARG 1 chunkPos
+	METHOD method_31811 removeIfEmpty (JLnet/minecraft/class_5572;)V
+		ARG 1 packedChunkSection
+		ARG 3 entities
+	METHOD method_31812 tryWriteEntityChunkData (JLjava/util/function/Consumer;)Z
```

Comment:
`tryWriteChunk`? I think the entity specifier might be unnecessary in a few of these functions

Reply:
It's for specifically writing entity data to the chunk. If I were to shorten it I would shorten it to `tryWriteEntityData`

Reply:
I imagine it's clear that it is related to entity storage given the class (and I assume it's an internal method?)